cypress/integration/e2e/atom.video.spec.js

```diff
@@ -0,0 +1,99 @@
+import { disableCMP } from '../../lib/disableCMP.js';
+
+const mainMediaVideo = 'https://www.theguardian.com/uk-news/2020/dec/04/edinburgh-hit-by-thundersnow-as-sonic-boom-wakes-residents'
+const mainMediaPlayEvent = {"id":"gu-video-youtube-2b33a7b7-e639-4232-9ecd-0fb920fa8147","eventType":"video:content:play"}
+
+const embedMediaVideo = 'https://www.theguardian.com/us-news/2020/oct/29/lordstown-ohio-trump-gm-plant-election'
+const embedPlayEvent = {"id":"gu-video-youtube-35f9a38f-c0c5-4bbd-9395-e338c9e97d30","eventType":"video:content:play"}
+
+describe('Video', function () {
+    describe('Main media youtube video', function () {
+        beforeEach(function () {
+            disableCMP();
+        });
+        it('should render', function() {
+            cy.visit(`/Article?url=${mainMediaVideo}`);
+            cy.get(`#youtube-block-main-media-0`).should('be.visible');
+        })
+
+        it('should hide overlay on click', function() {
```

Comment:
I think these tests could be better structured with fewer 'tests' but more assertions. See: https://docs.cypress.io/guides/references/best-practices.html#Creating-%E2%80%9Ctiny%E2%80%9D-tests-with-a-single-assertion

Generally, although this isn't a rule, there's a pattern where each test starts with a visit, and then some actions, and then the assertions. Having a test with no visit means it is dependant on previous tests, but we want each test to be modular and stand on it's own as much as possible.